Cubical/Structures/Ring.agda

```diff
@@ -151,10 +152,12 @@ record RingEquiv (R S : Ring {ℓ}) : Type ℓ where
 
 module RingΣTheory {ℓ} where
 
-  open Macro ℓ (recvar (recvar var) , var , recvar (recvar var)) public renaming
-    ( structure to RawRingStructure
-    ; equiv     to RawRingEquivStr
-    ; univalent to rawRingUnivalentStr )
+  RawRingStructure = λ (X : Type ℓ) → (X → X → X) × X × (X → X → X)
```

Comment:
Is this kind of change already done for other structures?

Reply:
Yeah, I just missed this one.